### ISSUE TITLE
Linuxbrew Updates

### ISSUE THREAD

**Issue by sjackman** [Linuxbrew/brew#1]
Subscribe to this issue to receive news about significant developments related to Linuxbrew.

**Comment by oldsharp** [comment]
Just curious, @sjackman does this issue means you are working on the migration from `linuxbrew` to `brew` and `homebrew-core`, just like the upstream did a few days ago?

**Comment by sjackman** [comment]
Sure are! You can test it out now, if you're the adventurous type.
```sh
ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install)"
```

**Comment by oldsharp** [comment]
Awesome!  Will try it out ;-)

**Comment by sjackman** [comment]
`brew update` will now migrate your legacy Linuxbrew installation to the new split repository.

**Comment by bw-yuhogg** [comment]
Installing formulae that depend on X11 (`depends_on :x11` in Ruby) will now automatically install the X.Org client libraries for you. That means you can now install X11 applications from Linuxbrew without depending on host libraries! (You still need a host X server, at least for now).

I've also bottled all of the default X11 dependencies to save you all the effort of compiling it yourself.

Big thanks to @maxim-belkin for packaging up the X11 libraries (available from https://github.com/linuxbrew/homebrew-xorg) and to @sjackman for helping me into the Homebrew/Linuxbrew community.

Note that there are still some outstanding issues related to using GTK+ and qt on Linuxbrew, but one step at a time :smile_cat: :heavy_multiplication_x:

**Comment by bw-yuhogg** [comment]
Linuxbrew can now install its own copy of Ruby 2.0 if your host system does not include Ruby 2.0. Support for Ruby 1.8 has been dropped from Linuxbrew. If you're using Linuxbrew on a system with a Ruby version < 2.0, no need to worry: Ruby 2.0 will be installed by `brew update`. This installation of Ruby 2.0 is designed for portability, thanks to the work of xu-cheng, so it works even if you're on an older distro.

**Comment by aghasemi** [comment]
Hi,
Is there any active development on the ChromeOS version?

**Comment by bw-yuhogg** [comment]
QT 5 has been fixed for Linuxbrew! A precompiled bottle is now available for it.

This will greatly improve the potential for GUI application distribution through Linuxbrew and is the result of much hard work by myself, the other Linuxbrew maintainers, and some external contributors. Particular credit goes to @donbright for fixing a major issue with the QT 5 build.

**Comment by bw-yuhogg** [comment]
Happy new year, Linuxbrew users!

2016 was an exciting year, with lots of changes to Linuxbrew that make it a more Linux-friendly, out-of-the-box compatible, and all-around great package manager.

* As Homebrew itself did, we split Linuxbrew into two repos: one for the package manager itself (brew) and another for the list of "core" formulae (homebrew-core), retaining the Git history of both along the way. Huge kudos to Shaun Jackman for this work and to the upstream Homebrew developers who did the same for themselves.

* We moved to our own GitHub organization, which enables us to create our own repositories much more freely.

* Homebrew announced a plan to support Linux at the brew level, albeit not at the core formula level (i.e. Homebrew/brew supports Linux, Homebrew/homebrew-core does not). Huge credit to Mike McQuaid for creating an abstraction layer that isolates Mac-specific and Linux-specific differences - it's cleaner and easier to maintain than Linuxbrew's code. (To be clear, we still recommend using Linuxbrew/brew for now)

* We added 3 new maintainers: myself, Piotr Gaczkowski, and  Maxim Belkin.

* We fixed a huge number of missing dependencies in low-level packages. Piotr in particular deserves a lot of credit for this work.

* We packaged the X.Org client libraries and related formulae like Wayland in the new homebrew-xorg tap. Massive props to Maxim for this effort.

* We fixed the GTK+ and Qt formulae, with GTK+3 just around the corner. Qt 5 has been an especially tricky formula to work with, but thanks to a massive effort from both inside and outside the maintainer circle we were able to greatly improve support for it.

* LLVM support has seen substantial improvements. Much credit both to Maxim and to various PR submitters and bug reporters from outside the maintainer circle for all the effort.

* We moved to a stable release model, with each release based on a stable release of upstream Homebrew plus our patches. 

What we'd like to accomplish in 2017:

* Start using Homebrew/brew as-is by the end of the year. This will be a big undertaking, but also a very exciting one.

* Add more maintainers :smile_cat:

* Migrate the default GCC to GCC 6. (We will likely upgrade to GCC 5.4 before tackling GCC 6)

* Build precompiled bottles for many more formulae.

* Provide Docker images combining major distros with Linuxbrew for easy usage. A Dockerfile based on Ubuntu 16.04 is already available from https://github.com/linuxbrew/docker, although it currently has to be built manually. We also hope to provide images for Debian and CentOS, among others. A Docker image based on Ubuntu 14 is currently available from Docker Hub: `docker run -it linuxbrew/linuxbrew` 

Big thanks to everyone who submitted PR's or bug reports over the last year.

**Comment by sjackman** [comment]
:champagne: Please welcome Jonathan Chang (@jonchang), the newest member of the Linuxbrew team of maintainers! Jon is "an evolutionary biologist whose research involves understanding the origin and maintenance of biological diversity".

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.1.11 has been released!
The default build environment for Linuxbrew is now `--env=super`! Superenv intercepts calls to the compiler by adding to the PATH a shell script named `cc`, which then calls the real compiler with the compiler options needed for that particular formula, like include and library paths. The previous standard build environment passes compiler options to the build tool using command line options and environment variables. The old standard build environment can be invoked using `--env=std`.

**Comment by sjackman** [comment]
🍾 Please welcome Michka Popoff (@iMichka), the newest member of the Linuxbrew team of maintainers! Michka develops software for atomic force microscopes, and he also helps maintain Homebrew/science.

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.2.2 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.2.2), from which Linuxbrew/brew is derived.

**Comment by sjackman** [comment]
💀 CentOS 5 has reached its end-of-life as of 2017-03-31. The precompiled binary bottles of Linuxbrew will no longer be supported on CentOS 5 after 2017-08-01. Installing formulae from source on CentOS 5 will be supported on a best-effort basis.

We currently build precompiled binary bottles on Ubuntu 14.04 LTS (Trusty Tahr) with glibc 2.19, which requires Linux kernel version 2.6.16 or later. CentOS 5 uses Linux kernel version 2.6.18.

Later this year, we plan to build binary bottles on Ubuntu 16.04 LTS (Xenial Xerus) with glibc 2.23, which requires Linux kernel version 2.6.32 or later. CentOS 5 with Linux kernel version 2.6.18 will thus no longer be able to use Linuxbrew bottles.

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.2.3 has been released!
This is my first merge as a new maintainer.

There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.2.3), from which Linuxbrew/brew is derived.

Enjoy :)

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.2.4 has been released!

There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.2.4) from which Linuxbrew/brew is derived.

Enjoy :)

**Comment by bw-yuhogg** [comment]
Anyone who's been having trouble with `stack level too deep` errors recently due to the migration of `hdf5` from homebrew-science to homebrew-core: your problems should be fixed now!

We apologize for the difficulty you've been experiencing in the meantime.

Big thanks to @maxim-belkin for his efforts to resolve this issue.

If you experience any further difficulties with this, please create a new issue in homebrew-core and \@mention me on it so we don't clutter the mailing list :)

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.2.5 has been released!

There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.2.5) from which Linuxbrew/brew is derived.

Enjoy :)

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.2.6 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.2.6), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
💡 The next release of Linuxbrew, 1.3.0, will require glibc 2.23 or greater to use precompiled binary bottles. If your host system does not provide a sufficiently new version of glibc, Linuxbrew will install a recent version of glibc. Users of Ubuntu 14.04 LTS (Trusty Tahr), who previously uses their host's glibc, will now use a brewed glibc. If you would like to continue using your host's glibc, consider upgrading to Ubuntu 16.04 LTS (Xenial Xerus).

**Comment by iMichka** [comment]
💡 We are pleased to announce that we have reached the symbolic milestone of 1,000 bottles (pre-compiled binary packages) in Linuxbrew/core, roughly 25% of the total possible. The 1,000 bottles mark has been achieved in roughly six months time, and new bottles are being added on a daily basis. You can benefit most from the bottles if you install Linuxbrew in `/home/linuxbrew/.linuxbrew` so that you can use bottles for non-relocatable formula. A new installation of Linuxbrew will use this directory by default if you have `sudo` access.

**Comment by sjackman** [comment]
🍾 Please welcome Alyssa Ross (@alyssais), the newest member of the Linuxbrew team of maintainers! Alyssa uses Linuxbrew on both Intel and Raspberry Pi ARM hardware and. She is also a maintainer of Homebrew for macOS. Welcome, Alyssa!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.2.8 has been released!  
You can read the release notes for the upstream [Homebrew/brew 1.3.0](https://github.com/Homebrew/brew/releases/tag/1.3.0), from which this release of Linuxbrew/brew 1.2.8 is derived.
`glibc` has been upgraded from version 2.19 to 2.23. The next planned release of the Docker image `linuxbrew/linuxbrew:1.3.0` will be based on Ubuntu 16.04 LTS (Xenial Xerus).
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.3.0 has been released!
You can read the release notes for the upstream release [Homebrew/brew 1.3.0](https://github.com/Homebrew/brew/releases/tag/1.3.0), from which Linuxbrew/brew is derived. The current Docker image `linuxbrew/linuxbrew:1.3.0` and `latest` is based on Ubuntu 16.04 LTS (Xenial Xerus). The previous Docker image `linuxbrew/linuxbrew:1.2.8` was based on Ubuntu 14.04 LTS (Trusty Tahr). The precompiled binary packages (known as bottles) are now being built using GCC 5.3 and glibc 2.23.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.3.1 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.3.1), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by bw-yuhogg** [comment]
I'm sorry to announce my departure as a maintainer of Linuxbrew, effective today.

I've really enjoyed my time as a maintainer, but feel I have to move on.

The project is in good hands and I have no doubts that the great community around Linuxbrew will continue to make it even better.

**Comment by sjackman** [comment]
It's been great working with you, Bob. We'll miss you. Your contribution to Linuxbrew was tremendous. All my best wishes! ❤️

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.3.2 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.3.2), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.3.3 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.3.3), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.3.4 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.3.4), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.3.5 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.3.5), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.3.6 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.3.6), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.3.7 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.3.7), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.3.8 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.3.8), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.3.9 has been released!
This release enables environment filtering (`HOMEBREW_ENV_FILTERING`) for those users running Linuxbrew in developer mode. Environment filtering will be enabled for all users with the upcoming release of Linuxbrew 1.4.0. Environment filtering helps enable reproducible builds by clearing the environment before building software.
You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.3.9), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.4.0 has been released!
This release enables environment filtering for all users. Environment filtering helps enable reproducible builds by clearing the environment before building software.
You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.4.0), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.4.1 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.4.1), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.4.2 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.4.2), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.4.3 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.4.3), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
ℹ️ Homebrew/science has been deprecated. The formulae in Homebrew/science have been archived at https://github.com/brewsci/homebrew-science. I've created a new tap for bioinformatics formulae at https://github.com/brewsci/homebrew-bio.
:beer: brew tap brewsci/bio
:beer: brew tap brewsci/science
:skull: brew untap homebrew/science

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.0 has been released!
This release deprecates several dependencies of the form `depends_on :requirement`, including `:ant :emacs :fortran :mpi :mysql :perl :postgresql :python :python3 :ruby :tex`. Dependencies may be used in place of requirements, such as `depends_on "openmpi"` rather than `depends_on :mpi`. You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.0), from which Linuxbrew/brew is derived.
Enjoy!

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.5.1 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.1) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.2 has been released!
This release fixes an issue where `X11Requirement` would not be satisfied even though `linuxbrew/xorg/xorg` is installed when `/usr/bin/Xorg` is old. Executables in poured bottles will not be affected by `LD_LIBRARY_PATH` by using `RPATH` rather than `RUNPATH`. You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.2) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.3 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.3) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.4 has been released!
This release avoids installing `bzip2`, `unzip`, `xz`, and `gpatch` if those tools are available in `/usr/bin`. You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.4) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
⚠️ Coming up soon, the Python 2 formula will be renamed from `python` to `python@2`, and the Python 3 formula will be renamed from `python3` to `python`. The `python` executable will be Python 3. The `python@2` formula is keg-only, meaning that it is not linked into your `PATH` by default. You can link it using `brew link --force python@2`, which will add a `python2` executable to your `PATH`.
For more information, see https://brew.sh/2018/01/19/homebrew-1.5.0/

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.5 has been released!
This release installs the `jdk` formula when a formula depends on Java, unless the host provides a sufficiently recent version of Java. You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.5) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.6 has been released!
This release installs the `unzip` formula when a formula needs it, unless the host provides `unzip`. You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.6) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.7 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.7) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.8 has been released!
This release fixes an issue with `brew linkage` incorrectly reporting dependencies in taps as undeclared. You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.8) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
⚠️ The Python 2 formula has been renamed from `python` to `python@2`, and the Python 3 formula has been renamed from `python3` to `python`. The `python` executable is now Python 3. The `python@2` formula is keg-only, meaning that it is not linked into your `PATH` by default. You can link it using `brew link --force python@2`, which will add a `python2` executable to your `PATH`.
For more information, see https://brew.sh/2018/01/19/homebrew-1.5.0/

**Comment by sjackman** [comment]
ℹ️ The `python` executable is once again Python 2. The `python@2` formula is no longer keg-only, so the brewed `python` will be found in your `PATH`.
For more information, see https://docs.brew.sh/Homebrew-and-Python

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.9 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.9) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.10 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.10) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.11 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.11) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.12 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.12) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.13 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.13) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.5.14 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.5.14) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.0 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.0) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.1 has been released!
The compiler `gcc-5` is selected preferentially over more recent versions of GCC. You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.1) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.2 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.2) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.3 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.3) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.4 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.4) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.5 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.5) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.6 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.6) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.7 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.7) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.8 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.8) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.9 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.9) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.10 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.10) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.11 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.11) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.6.12 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.12) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.6.13 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.13) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.6.14 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.14) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.6.15 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.15) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.6.16 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.16) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.6.17 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.6.17) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.7.0 has been released!
You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.7.0) from which Linuxbrew/brew is derived.

⚠️ From this release on, `glibc` 2.12 or newer is required to use Linuxbrew.

Enjoy!

**Comment by iMichka** [comment]
🎉 Linuxbrew 1.7.1 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.7.1) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.7.2 has been released!
This release fixes a bug that caused installing `glibc` from a precompiled binary bottle to fail. You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.7.2) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.7.3 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.7.3) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.7.4 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.7.4) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.7.5 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.7.5) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.7.6 has been released!
This release resolves the bug producing the error message `cannot load such file -- plist (LoadError)`. You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.7.6) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.7.7 has been released!
This release resolves the bug producing the error message `ArgumentError: parent directory is world writable but not sticky` on Windows Subsystem for Linux (WSL). You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.7.7) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.8.0 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the [release notes](https://brew.sh/2018/10/23/homebrew-1.8.0/) for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.8.0) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.8.1 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.8.1) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.8.2 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.8.2) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.8.3 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.8.3) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.8.4 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.8.4) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.8.5 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.8.5) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.8.6 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.8.6) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.9.0 has been released!
[Homebrew 1.9.0](https://brew.sh/2019/01/09/homebrew-1.9.0/) has beta support for Linux and Windows 10 (with the Windows Subsystem for Linux). Homebrew 2.0.0 will add full support for Linux. Linuxbrew 1.9.x will be the final release of Linuxbrew. Linuxbrew users will be migrated automatically to Homebrew. You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.9.0) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.9.1 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.9.1) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.9.2 has been released!
There are no release notes specific to Linuxbrew for this release, but you can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.9.2) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.9.3 has been released!
Linuxbrew/brew has been merged into Homebrew/brew! Linuxbrew/brew will no longer be updated. Run `brew doctor` to see instructions to migrate your current installation of Linuxbrew from Linuxbrew/brew to Homebrew/brew. The next release of Linuxbrew will migrate you automatically to Homebrew/brew.
You can read the release notes for the upstream [Homebrew/brew](https://github.com/Homebrew/brew/releases/tag/1.9.3) from which Linuxbrew/brew is derived.
Enjoy!

**Comment by sjackman** [comment]
🎉 Linuxbrew 1.9.4 has been released!
Linuxbrew/brew has been merged into [Homebrew/brew](https://github.com/Homebrew/brew)! Existing installations of Linuxbrew will be automatically migrated to Homebrew. Linuxbrew/brew will no longer be updated. See the [Homebrew documentation of Linuxbrew](https://docs.brew.sh/Linuxbrew) and the Homebrew 2.0.0 [blog post](https://brew.sh/2019/02/02/homebrew-2.0.0/).
Enjoy!
